scripts/webpack/webpack.dev.js

```diff
@@ -81,7 +81,7 @@ module.exports = (env = {}) =>
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, '../../public/views/index.html'),
         template: path.resolve(__dirname, '../../public/views/index-template.html'),
-        inject: 'body',
+        inject: false,
```

Comment:
what does this has to do with collecting frontend metrics?

Reply:
Please take a look at `index-template.html`https://github.com/grafana/grafana/pull/20365/files#diff-34712c43373283c700a1c3d6ff0c9d31 file changes, especially the very bottom of it https://github.com/grafana/grafana/pull/20365/files#diff-34712c43373283c700a1c3d6ff0c9d31R312. 

Instead of relying on HtmlWebpackPlugin's injection I am manually rendering the JS chunks so that performance mark (js done blocking) can be added AFTER the scripts are parsed and executed.

Reply:
I see!